error building Sandstorm in Ubuntu 15.10(beta)

```
install: sandstorm/sandstorm.ekam-manifest
  sandstorm/run-bundle: not found
link: sandstorm/run-bundle.o
  tmp/kj/async-io.o: In function `kj::Function<void ()>::Impl<kj::CaptureByMove<kj::(anonymous namespace)::SocketAddress::lookupHost(kj::LowLevelAsyncIo
    Provider&, kj::String, kj::String, unsigned int)::$_6, kj::(anonymous namespace)::SocketAddress::LookupParams> >::operator()()':
  /ekam-provider/canonical/kj/async-io.c++:(.text+0xd3b7): warning: Using 'getaddrinfo' in statically linked applications requires at runtime the shared
    libraries from the glibc version used for linking
  tmp/sandstorm/run-bundle.o: In function `sandstorm::RunBundleMain::getMain()':
  /ekam-provider/canonical/sandstorm/run-bundle.c++:(.text._ZN9sandstorm13RunBundleMain7getMainEv[_ZN9sandstorm13RunBundleMain7getMainEv]+0x84): 
    undefined reference to `sandstorm::SupervisorMain::SupervisorMain(kj::ProcessContext&)'
  tmp/sandstorm/run-bundle.o: In function `sandstorm::SupervisorMain::~SupervisorMain()':
  /ekam-provider/canonical/sandstorm/run-bundle.c++:(.text._ZN9sandstorm14SupervisorMainD2Ev[_ZN9sandstorm14SupervisorMainD2Ev]+0xb): undefined 
    reference to `vtable for sandstorm::SupervisorMain'
  clang: error: linker command failed with exit code 1 (use -v to see invocation)
compile: sandstorm/supervisor.c++
  In file included from /ekam-provider/canonical/sandstorm/supervisor.c++:41:
  In file included from /ekam-provider/c++header/sandstorm/ip_tables.h:21:
  In file included from /usr/include/linux/netfilter_ipv4.h:8:
  In file included from /usr/include/linux/netfilter.h:7:
  /usr/include/linux/in.h:28:3: error: redefinition of enumerator 'IPPROTO_IP'
    IPPROTO_IP = 0,               /* Dummy protocol for TCP               */
    ^
  /usr/include/netinet/in.h:43:21: note: expanded from macro 'IPPROTO_IP'
  #define IPPROTO_IP              IPPROTO_IP
                                  ^
  /usr/include/netinet/in.h:42:5: note: previous definition is here
      IPPROTO_IP = 0,        /* Dummy protocol for TCP.  */
      ^
  In file included from /ekam-provider/canonical/sandstorm/supervisor.c++:41:
  In file included from /ekam-provider/c++header/sandstorm/ip_tables.h:21:
  In file included from /usr/include/linux/netfilter_ipv4.h:8:
  In file included from /usr/include/linux/netfilter.h:7:
  /usr/include/linux/in.h:30:3: error: redefinition of enumerator 'IPPROTO_ICMP'
    IPPROTO_ICMP = 1,             /* Internet Control Message Protocol    */
    ^
  /usr/include/netinet/in.h:45:23: note: expanded from macro 'IPPROTO_ICMP'
  #define IPPROTO_ICMP            IPPROTO_ICMP
                                  ^
  /usr/include/netinet/in.h:44:5: note: previous definition is here
      IPPROTO_ICMP = 1,      /* Internet Control Message Protocol.  */
      ^
  In file included from /ekam-provider/canonical/sandstorm/supervisor.c++:41:
  In file included from /ekam-provider/c++header/sandstorm/ip_tables.h:21:
  In file included from /usr/include/linux/netfilter_ipv4.h:8:
  In file included from /usr/include/linux/netfilter.h:7:
  ...(log truncated; use -l to increase log limit)...
```

This also looks like a significant set of lines:

```
  In file included from /ekam-provider/c++header/sandstorm/ip_tables.h:21:
  In file included from /usr/include/linux/netfilter_ipv4.h:8:
  In file included from /usr/include/linux/netfilter.h:7:
  /usr/include/linux/in.h:28:3: error: redefinition of enumerator 'IPPROTO_IP'
    IPPROTO_IP = 0,               /* Dummy protocol for TCP               */
```

--- a/src/sandstorm/supervisor.c++
+++ b/src/sandstorm/supervisor.c++
@@ -24,6 +24,7 @@
 #include <capnp/rpc-twoparty.h>
 #include <capnp/rpc.capnp.h>
 #include <unistd.h>
+#include <netinet/in.h> // needs to be included before sys/capability.h
 #include <sys/stat.h>
 #include <sys/types.h>
 #include <sys/time.h>
@@ -35,7 +36,6 @@
 #include <sys/capability.h>
 #include <sys/ptrace.h>
 #include <sys/syscall.h>
-#include <netinet/in.h>
 #include <linux/sockios.h>
 #include <linux/route.h>
 #include <sandstorm/ip_tables.h>  // created by Makefile from <linux/netfilter_ipv4/ip_tables.h>

